src/plone/app/multilingual/browser/menu.py

```diff
@@ -221,7 +221,7 @@ def getMenuItems(self, context, request):
                 "selected": False,
                 "icon": None,
                 "extra": {
-                    "id": "_shared_folder",
```

Comment:
just a passing correction - IIUC it becomes the CSS id of the menu item, which could (potentially) be used in testing.  Looking at other precedents in the file, it's name should be the name of title with an initial underscore.  But in this case (due to a cut& paste error) it doesn't match.  I could leave it out since it's not relevant, but I could also keep it in since it's not referenced anywhere else at the moment.  Just say & I drop it!